aws-analytics-pinpoint/src/test/java/com/amplifyframework/analytics/pinpoint/EventRecorderTest.kt

```diff
@@ -0,0 +1,148 @@
+package com.amplifyframework.analytics.pinpoint
+
+import android.database.MatrixCursor
+import androidx.test.core.app.ApplicationProvider
+import aws.sdk.kotlin.services.pinpoint.PinpointClient
+import aws.sdk.kotlin.services.pinpoint.model.EventItemResponse
+import aws.sdk.kotlin.services.pinpoint.model.EventsResponse
+import aws.sdk.kotlin.services.pinpoint.model.ItemResponse
+import aws.sdk.kotlin.services.pinpoint.model.PutEventsRequest
+import aws.sdk.kotlin.services.pinpoint.model.PutEventsResponse
+import com.amplifyframework.analytics.pinpoint.database.EventTable
+import com.amplifyframework.analytics.pinpoint.database.PinpointDatabase
+import com.amplifyframework.analytics.pinpoint.models.AndroidAppDetails
+import com.amplifyframework.analytics.pinpoint.models.AndroidDeviceDetails
+import com.amplifyframework.analytics.pinpoint.models.PinpointEvent
+import com.amplifyframework.analytics.pinpoint.models.PinpointSession
+import com.amplifyframework.analytics.pinpoint.models.SDKInfo
+import com.amplifyframework.analytics.pinpoint.targeting.TargetingClient
+import com.amplifyframework.analytics.pinpoint.targeting.endpointProfile.EndpointProfile
+import io.mockk.coEvery
+import io.mockk.coVerify
+import io.mockk.coVerifyOrder
+import io.mockk.mockk
+import java.util.UUID
+import kotlinx.coroutines.CoroutineDispatcher
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.test.UnconfinedTestDispatcher
+import kotlinx.coroutines.test.runTest
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+
+/*
+ * Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
```

Comment:
move to the top of the page

Reply:
This file will be deleted in the following PR so probably wont change this.

Reply:
This file will be deleted in the next PR.